README.md

```diff
@@ -1,59 +1,168 @@
-# TODO.md
+# Description
+[EN] Repositorie about how to do TO-DO list on markdown.
+
+# Content EN
+- [Structure of directories](#Structure-of-directories)
+- [What is TODO.md?](#What-is-`TODO.md`?)
+- [Why keep a TODO.md?](#Why-keep-a-`TODO.md`?)
+- [Why not using online management tools?](#Why-not-using-online-management-tools?)
+- [TODO.md format](#`TODO.md`-format)
+- [Examples](#examples)
+- [See also](#See-also)
+
+# Contenido ES
+- [Estructura de los directorios](#Estructura-de-los-directorios)
+- [¿Qué es un TODO.md?](#¿qué-es-un-`TODO.md`?)
+- [¿Por qué dejar un TODO.md?](#¿por-qué-dejar-un-`TODO.md`?)
+- [¿Por qué no usar una herramienta online?](#¿Por-qué-no-usar-una-herramienta-online?)
+- [Formato del TODO.md](#formato-del-`TODO.md`)
+- [Ejemplos](#ejemplos)
+- [Ver también](#ver-también)
+----
+## Structure of directories
+```txt
+.
+├── README.md
+├── LICENSE
+└── Examples_TODO
+    └── TODO.md
+    └── TODO2.md
+    └── TODO3.md
+    └── TODO[..].md
+```
 
-### What is TODO.md?
+## What is `TODO.md`?
 
-The TODO.md format is based on [GFM - GitHub Flavored Markdown - Task Lists](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown).
+The `TODO.md` format is based on [GFM - GitHub Flavored Markdown - Task Lists](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown).
 
-TODO.md is a file that contains tasks organized in multiple sections.
+`TODO.md` is a file that contains tasks organized in multiple sections.
 
-Tasks in TODO.md can be visualized using [Kanban Board](https://en.wikipedia.org/wiki/Kanban_board) where sections become columns on the board.
+Tasks in `TODO.md` can be visualized using [Kanban Board](https://en.wikipedia.org/wiki/Kanban_board) where sections become columns on the board.
 
-### Why keep a TODO.md?
+## Why keep a `TODO.md`?
 
 For a project, there any many sources of information like Documentation, Issue forum, Wiki pages, etc. It is hard to find out about the current progress.
 
-Keeping a TODO.md file makes it easier for anyone wants to know about the project's plans and work needs to be done.
+Keeping a `TODO.md` file makes it easier for anyone wants to know about the project's plans and work needs to be done.
 
-### Why not using online management tools?
+## Why not using online management tools?
```

Comment:
You should fix this typo: `## Why not using online management tools?` should be `## Why are we not using online management tools?`.